packages/ui-map-components/src/components/Table/getMapTableData.tsx

```diff
@@ -26,9 +26,9 @@ const processColumns = (serieses: Series[]) => {
       Header: 'Name',
       accessor: 'name',
       // eslint-disable-next-line react/prop-types
-      Cell: ({ value }: { value: string | number | boolean | undefined }) => (
-        <FirstColumnCell>{String(value)}</FirstColumnCell>
-      ),
+      Cell: ({ value }: { value: string | number | boolean | undefined }) => {
```

Comment:
Just need to replace the curly braces here with regular parentheses, otherwise nothing is getting returned in this `Cell` function